ls1intum/Artemis
# `Exam mode`: Fix translations

### Checklist
#### General


#### Client

### Motivation and Context
Exam translations were pretty broken because the JSON structure did not match the keys used in the code. Additionally, the create test runs dialogue was missing translations.

### Description
This PR fixes the structure of the JSON files and adds translation to the dialogue.

### Steps for Testing
1. Create an exam as instructor and check the various exam management pages for translation errors.

### Review Progress
#### Code Review (when the translations are shown correctly, then the code changes worked)
#### Manual Tests

### Test Coverage
unchanged

### Screenshots
<details>
<summary>Click to Unfold</summary>

Before:
![Screenshot_exam_before](https://user-images.githubusercontent.com/64250573/143046352-817b6ff0-5ca8-446e-aacc-7da4c696dd95.png)
![create_test_run_before](https://user-images.githubusercontent.com/64250573/143046348-cf809dc5-a8c7-484f-a8e6-1de2474a69e4.png)

After:
![Screenshot_exam_after](https://user-images.githubusercontent.com/64250573/143046219-583de998-2ebc-4024-9b30-a33e10776665.png)
![test_run](https://user-images.githubusercontent.com/64250573/143046225-70e1a2fb-7b16-4f03-8bdc-5f8f695c05b9.png)
</details>
